packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/SampleModuleCPP.h

```diff
@@ -33,7 +33,13 @@ struct SampleModuleCppImpl {
 #pragma region Initialization
 
   REACT_INIT(Initialize)
-  void Initialize(ReactContext const & /*reactContext*/) noexcept {
+  void Initialize(ReactContext const &reactContext) noexcept {
+    PropertyName<int> myProp1{L"Prop1"};
+    PropertyName<winrt::hstring> myProp2{L"Prop2"};
+
+    DEBUG_OUTPUT("globalProps.Prop1:", *reactContext.GlobalProperties().Get(myProp1));
```

Comment:
>Get(myProp1) [](start = 72, length = 12)

Can't tell from the usage context: Is this a strongly-typed Get?

Reply:
Yes, the myProp1 encapsulates the name and the type.